scripts/cmake-configurator/parser/test_cmakerow.py

```diff
@@ -0,0 +1,44 @@
+from unittest import TestCase
+from cmake_row import cmake_row
+
+class test_cmake_cache_variable(TestCase):
+   commentline = ""
+   filepath_var_valid = "CMAKE_ADDR2LINE:FILEPATH=/usr/bin/addr2line"
+   noname_var = ":FILEPATH=/usr/bin/addr2line"
+   notype_var = "CMAKE_ADDR2LINE:=/usr/bin/addr2line"
+
+   def test_row_creation(self):
+      var_name = "CMAKE_ADDR2LINE"
+      row = cmake_row(self.filepath_var_valid, 0)
+      with self.subTest():
+         self.assertEqual(row.name, var_name)
+      with self.subTest():
+         self.assertEqual(row.var_type, "FILEPATH")
+      with self.subTest():
+         self.assertEqual(row.type, row.VARIABLE)
+      with self.subTest():
+         self.assertEqual(row.val,  "/usr/bin/addr2line")
+      with self.subTest():
+         self.assertEqual(row.row_number, 0)
+
+   def test_slash_comment_row(self):
```

Comment:
That's cool, but where do we check the contents of the "comment" field? The idea is that in the next step the comments will be very important for us, because they will contain metadata. So, could you compare the value of "comment" with some expected value? Note: don't forget the multi-lib comment case which is essential.

Reply:
Done